timemachine/cpp/src/wrap_kernels.cpp

```diff
@@ -167,7 +167,10 @@ void declare_context(py::module &m) {
         py::array_t<double, py::array::c_style> out_x_buffer({F, N, D});
         std::memcpy(out_x_buffer.mutable_data(), result[1].data(), result[1].size()*sizeof(double));
 
-        return py::make_tuple(out_du_dl_buffer, out_x_buffer);
+        py::array_t<double, py::array::c_style> box_buffer({D, D});
```

Comment:
Question: do we want this to be of shape `{F, D, D}` (store box vectors at every frame for which we return coordinates), or `{D, D}` (final box vectors only)?

Reply:
+1 - we need this to be `{F, D, D}`.

Reply:
In particular, `F` should be identical to the `F` in out_x_buffer

Reply:
Nicely resolved in https://github.com/proteneer/timemachine/pull/444/commits/66f1375e0c73256a7dea74a59d371cd803ca3b4d